### DISCUSSION TITLE
使用MPV_lazy播放非常卡顿

### DISCUSSION THREAD

**Discussion by bricle** [hooke007/mpv_PlayKit#112] [Q&A 问答] [answered]
视频是Interstellar.2014.IMAX.UHD.BluRay.2160p.10bit.HDR.2Audio.DTS-HD.MA.5.1.TrueHD.5.1.x265.mkv
使用mpv，mpv.net，mpvnet(cm)都不会卡顿
以上均使用默认播放设置

**Comment by hooke007** [comment, marked as answer]
我能想到的主要潜在因素可能是这2个，尝试禁用
https://github.com/hooke007/MPV_lazy/blob/01b991f1863c44d51e04e4c2f94fcc29168464e6/portable_config/mpv.conf#L23
https://github.com/hooke007/MPV_lazy/blob/01b991f1863c44d51e04e4c2f94fcc29168464e6/portable_config/mpv.conf#L149

**Reply by bricle** [reply]
非常感谢，只禁用了前两个已经不卡顿了
谢谢大佬orz

**Reply by hooke007** [reply]
。。。你可以尝试只禁用1或2看看具体是谁主因，当然本质都是硬件性能不足

**Comment by bricle** [comment]
哈哈，我设备是拯救者Y7000,1050显卡+i5 8300H，
 只把hwdec  改成nvdec-copy 就不卡了
